src/screens/bottomTabs/Wishlist/Wishlist.tsx

```diff
@@ -50,7 +56,8 @@ const Wishlist = ({ navigation }: Props) => {
           renderItem={({ item }) => (
             <WishlistItem
               wishlistDetails={item}
-              onDelete={() => handleDelete(item.accommodation.id)}
+              onDelete={() => handleRemoveFromWishlist(item.accommodation.id)}
```

Comment:
https://github.com/VentionCapstone/Mobile/pull/18/files#diff-e1ebb7ba104a124413913dd9589d9f6a5324b703c9f8c146f774234d0528f4f0L14

You can add `accomodationId` argument to `onDelete` callback prop to avoid passing inline functions here